src/modules/user-settings/lib/win-editor-path.ts

```diff
@@ -1,5 +1,6 @@
 import fs from 'fs';
 import nodePath from 'path';
+import _ from 'lodash';
```

Comment:
I am not a fan of lodash, sicne at my previous job, we had some pain around this lib. It's mostly personal opinion, since in general this lib is good.
So if we proceed with including it in our codebase, I would advocate for installing not the whole lib, but just specific functions, to avoid potential issues with tree shaking - https://www.npmjs.com/package/lodash.escaperegexp

Reply:
Update after discussion here p1747666104280689-slack-C06DRMD6VPZ:

At least, let's import the package directly from lodash, to improve tree-shaking:
`import escapeRegExp from 'lodash.escapeRegExp';`